src/lichess.js

```diff
@@ -75,7 +75,7 @@ var makeRequest = (function() {
     }
     processRequest();
     return makeRequest;
-})();
+}());
```

Comment:
Looking at the rule, it seems that either is the same and this is purely a stylistic choice.  http://eslint.org/docs/2.0.0/rules/wrap-iife.  We can set it to inside, outside, or any.  I can change the rule and revert it if you would rather have outside.  I don't have a preference either way.